src/Microsoft.ML.Data/Commands/DataCommand.cs

```diff
@@ -46,7 +46,8 @@ public abstract class ArgumentsBase
             public bool? Verbose;
 
             [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, HelpText = "The web server to publish the RESTful API", Hide = true)]
-            public ServerChannel.IServerFactory Server;
+            [BestFriend]
+            internal ServerChannel.IServerFactory Server;
```

Comment:
>internal [](start = 12, length = 8)

it's part of nested class in internal class. It's already invisible. You don't have to do that.